cmd/spire-server/cli/run/run.go

```diff
@@ -323,6 +324,8 @@ func newServerConfig(c *config) (*server.Config, error) {
 		sc.CATTL = ttl
 	}
 
+	sc.JWTIssuer = c.Server.JWTIssuer
```

Comment:
Assuming it's not too difficult to do, would be good to get a small test case to ensure this remains wired in. I've been bitten (and seen others get bitten) in the past during refactors etc by missing these one-liners.

Reply:
Oh, yep. Tests. Will do!